_sass/_layout.scss

```diff
@@ -0,0 +1,17 @@
+/**
+ * Summary
+ */
+ .summary {
+   h1 > a {
+     color: #ffffff;
+     text-decoration: none;
+   }
+ }
+
+ /**
+  * Get Started
+  */
+  .get-started {
+    padding: 50px;
+    margin: 20px;
```

Comment:
Donner des dimensions en [`em`, pas en `px`](http://stackoverflow.com/questions/609517/why-em-instead-of-px).